src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,57 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  title: string,
+  description?: string,
+  imgUrl: string,
+  imdbUrl: string,
+};
```

Comment:
```suggestion
type Props = {
  movie: Movie;
};
```
You have the interface `Movie`. Let's use it here